ts/components/FiscalCodeLandscapeOverlay.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * A component to show the fiscal code fac-simile in Landscape
+ */
+import { Body, Button, Container, Content, View } from "native-base";
+import * as React from "react";
+import { Platform, StatusBar, StyleSheet } from "react-native";
+import { isIphoneX } from "react-native-iphone-x-helper";
+import { UserProfile } from "../../definitions/backend/UserProfile";
+import IconFont from "../components/ui/IconFont";
+import customVariables from "../theme/variables";
+import FiscalCodeComponent from "./FiscalCodeComponent";
+import AppHeader from "./ui/AppHeader";
+
+type Props = Readonly<{
+  onCancel: () => void;
+  profile: UserProfile;
+  showBackSide?: boolean;
+}>;
+
+const globalHeaderHeight: number = Platform.select({
+  ios: isIphoneX()
```

Comment:
Just realized we can do it better, sorry ;)
```suggestion
  ios: customVariables.appHeaderHeight + (isIphoneX() ? 42 : 18),
```

Reply:
right, updated!